### ISSUE TITLE
Webtorrent: Super Peer

### ISSUE THREAD

**Issue by mitra42** [internetarchive/dweb-transport#23]
Moving discussion on the super peer to here. 

Outline: 
Tracker (in /Tracker) optionally adds Super Peers to its list
Super Peers respond to a BTIH by checking with the archive server to see if we have it, and if so read from the server (using standard WT http fallback).
This is partly complete by John Heisey

**Comment by mitra42** [comment]
See also https://github.com/internetarchive/dweb-transport/issues/23

**Comment by mitra42** [comment]
First version appears to be working in the test.   Next step (for @jhiesey ) is ensuring it works under WS/WebRTC with SuperPeer.

**Comment by mitra42** [comment]
@jhiesey I fixed the issue where it was getting the unmodified torrent.

**Comment by mitra42** [comment]
@jhiesey - the current version doesnt run the seeder because it can't find wrtc, I noticed wrtc isn't in package.json.  I'd add it but I remember you said there were two options (electron and some other) and I'm not sure which you ended up using. 

Can you add whichever you want and I'll install.

**Comment by jhiesey** [comment]
It is in the version of package.json I had: https://github.com/jhiesey/dweb-transport/blob/webtorrent-seed/package.json

Looks like something weird happened in the git history since then?

**Comment by jhiesey** [comment]
The other option is `electron-webrtc` which is included as a dev dependency and used for the tests since I've found it slightly more reliable and it's easy to run on a non-server.

**Comment by mitra42** [comment]
yes - something odd happened in the git history, and I just tried to restore it, but I couldn't find wrtc in anything. There is no directory webtorrent-seed so I must be missing something.

**Comment by mitra42** [comment]
yes - something odd happened in the git history, and I just tried to restore it, but I couldn't find wrtc in anything, and there is no directory webtorrent-seed in https://github.com/internetarchive/dweb-transport

**Comment by mitra42** [comment]
Ok - looks like I figured it out ... my mistake - conflicted package.json from two different branches, I'll fix it.

**Comment by mitra42** [comment]
It appears to be running - I posted a question on the Pull Request about testing it.
